src/api/services/adminServiceHelpers.ts

```diff
@@ -57,11 +57,23 @@ export const AllowedSiteRoles: Record<string, AvailableClientRole[]> = {
   [ParticipantTypeData.Publisher.typeName]: ['GENERATOR', 'SHARER'],
 };
 
+export const AllowedSiteRolesById: Record<number, number[]> = {
+  1: [3],
```

Comment:
Maybe a comment in code to explain these? 
I guess it means
```
  1: [3], // DSP: ID_READER
  2: [1], // Advertiser: MAPPER
  3: [1], // Data Provider: MAPPER
  4: [2], // Publisher: GENERATOR
```

Reply:
Good idea.  I added the comments.